addons/main/CfgVehicles.hpp

```diff
@@ -40,4 +40,13 @@ class CfgVehicles {
         function = QFUNC(moduleHeal);
         icon = "\A3\ui_f\data\Map\VehicleIcons\pictureHeal_ca.paa";
     };
+
+    class Camping_base_F;
+    class Land_MedicalTent_01_base_F : Camping_base_F {
+        class EventHandlers {
+            class CBA_Extended_EventHandlers {
+                init = "call cba_xeh_fnc_init";
```

Comment:
Do you only want to enable the Init XEH of this object? This might break other mods that also do something here. the CBA namespace is usually off limits for reasons like this

Reply:
Yeah, got too focused on CBA stuff when I could've done the vanilla thing there.